Sources/Orbit/Support/Alert/AlertContent.swift

```diff
@@ -0,0 +1,187 @@
+        icon
+            .font(.system(size: Orbit.Icon.Size.normal.value))
+            .foregroundColor(iconColor ?? outlineColor)
+            .iconColor(iconColor ?? outlineColor)
+            .accessibility(.alertIcon)
+    }
+
+    @ViewBuilder var titleContent: some View {
+        Text(title)
+            .bold()
+            .accessibility(.alertTitle)
+    }
+
+    @ViewBuilder var background: some View {
+        backgroundColor
+            .overlay(
+                RoundedRectangle(cornerRadius: BorderRadius.default)
+                    .strokeBorder(strokeColor, lineWidth: 1)
+            )
+            .overlay(
+                outlineColor
+                    .frame(height: 3),
+                alignment: .top
+            )
+    }
+
+    var outlineColor: Color {
+        defaultStatus.color
+    }
+
+    var backgroundColor: Color {
+        isSubtle ? .cloudLight : defaultStatus.lightColor
+    }
+
+    var strokeColor: Color {
+        isSubtle ? .cloudLightHover : defaultStatus.lightHoverColor
+    }
+
+    var defaultStatus: Status {
+        status ?? .info
+    }
+}
+
+// MARK: - Identifiers
+public extension AccessibilityID {
+    static let alertTitle           = Self(rawValue: "orbit.alert.title")
+    static let alertIcon            = Self(rawValue: "orbit.alert.icon")
+    static let alertDescription     = Self(rawValue: "orbit.alert.description")
+    static let alertButtonPrimary   = Self(rawValue: "orbit.alert.button.primary")
+    static let alertButtonSecondary = Self(rawValue: "orbit.alert.button.secondary")
+}
+
+struct AlertContentPreviews: PreviewProvider {
+
+    static let title = "Title"
+    static let description = """
+        The main description message of this Alert component should be placed here. If you need to use TextLink \
+        in this component, please do it by using <a href="..">Normal Underline text style</a>.
+
+        Description message can be <strong>formatted</strong>, but if more <ref>customizaton</ref> is needed a custom \
+        description content can be provided instead.
+        """
+
+    static var previews: some View {
+        PreviewWrapper {
+            content
+            inline
+        }
+    }
+
+    static var content: some View {
+        AlertContent(
+            title: title,
+            description: description,
+            isInline: false,
+            isSubtle: false
+        ) {
+            contentPlaceholder
+        } icon: {
+            Icon(.informationCircle)
+        } buttons: {
+            Button("Primary") {}
+        }
+        .previewDisplayName()
+    }
+
+    static var inline: some View {
+        AlertContent(
+            title: title,
+            description: description,
+            isInline: true,
+            isSubtle: false
+        ) {
+            contentPlaceholder
+        } icon: {
+            Icon(.informationCircle)
+        } buttons: {
+            Button("Primary") {}
```

Comment:
The preview shows default buttons - maybe intentional, but a `.buttonStyle` could be added to show the example fully customized.

<img width="372" alt="image" src="https://github.com/kiwicom/orbit-swiftui/assets/35844477/da831d3e-3712-4301-a3ed-239356f2a6f5">

Reply:
**Any**  button style or the one that should be there?

Reply:
I'll use the "correct" one.